src/main/resources/uk/gov/ida/rp/testrp/views/samlRedirectView.ftl

```diff
@@ -50,4 +39,5 @@
     <button class='verify-button' id="continue-button">Continue</button>
 </form>
 </body>
+<script type="text/javascript" src="/assets/scripts/saml-redirect-auto-submit.js"></script>
```

Comment:
We should keep this in the head and mark the `script` tag with a `defer` attribute instead. That'll get fetched asyncronously and then executed after page load, so it'll still have the `'load'` event triggered.

```html
<script type="text/javascript" src="/assets/scripts/saml-redirect-auto-submit.js" defer></script>
```